docs/routing-rules.md

```diff
@@ -6,13 +6,72 @@ By default, Trino Gateway reads the `X-Trino-Routing-Group` request header to
 route requests. If this header is not specified, requests are sent to default
 routing group (adhoc).
 
-The routing rules engine feature enables you to write custom logic to route
+The routing rules engine feature enables you to either write custom logic to route
 requests based on the request info such as any of the [request
-headers](https://trino.io/docs/current/develop/client-protocol.html#client-request-headers).
+headers](https://trino.io/docs/current/develop/client-protocol.html#client-request-headers),
+or set a URL address to make HTTP POST request and route based on returned result. 
+
 Routing rules are separated from Trino Gateway application code to a
-configuration file, allowing for dynamic rule changes.
+configuration file and url, allowing for dynamic rule changes.
+
+### Enabling routing rules engine
+
+To enable routing rules engine, find the following lines in `gateway-ha-config.yml`.
+Set `rulesEngineEnabled` to True, then `rulesType` as `FILE` or `REST_API`.
+Then either add `rulesConfigPath` to the path to your rules config file or set `rulesRESTConfiguration`
+to requesting url. `rulesType` is by default `FILE` unless specified.
+
+```yaml
+routingRules:
+  rulesEngineEnabled: true
+  rulesType: FILE
+  rulesConfigPath: "src/test/resources/rules/routing_rules.yml" # replace with path to your rules config file
+  rulesRESTConfiguration:
+    urlPath: localhost:8080/api/public/gateway_rules # replace with your own API path
```

Comment:
should this not be a separate server .. so maybe use router.example.com/gateway-rules or so? Also doesnt it need http:// or https://